adoc/deployment-sysreqs.adoc

```diff
@@ -156,6 +156,30 @@ The management workstation needs at least the following networking permissions:
 * Access to the `apiserver` (the load balancer should expose it, port `6443`), that will in turn talk to any master in the cluster
 * Access to Dex on the configured `NodePort` (the load balancer should expose it, port `32000`) so when the OIDC token has expired, `kubectl` can request a new token using the refresh token
 
+==== Sub-Network Sizing
+
+[IMPORTANT]
+====
+The service subnet and pod subnet must not overlap.
+
+Please plan generously for workload and the expected size of the networks before bootstrapping.
+====
+
+The default pod subnet is `10.244.0.0/16`. It allows for 65536 IP addresses overall.
+Assignment of CIDR's is by default `/24` (256 IP addresses per node) per node.
```

Comment:
256 I think is a little misleading. hopefully anyone doing this will understand that there are only 254 usable IP's in that range. I might reword this to `(254 usable IP addressed per node)` or something like this as this is what people would really care about anyhow. [1]